onnxruntime/python/tools/transformers/benchmark_gpt2_LM/README.md

```diff
@@ -0,0 +1,100 @@
+**Benchmark tool**
+
+This can be used to test beam search on any gpt2 model. Both pytorch and onnx models can be tested.
+
+Other kind of models are not supported yet.
+
+Also, this script defaults to GPU. To test a model on CPU pass --device 'cpu' when running the model.
+
+
+
+**Requirements:**
+
+1. GPU machine. V100 was used to test this. 
+
+2. GPT2 ONNX Model that follows the regular GPT2 style of inputs and outputs.
+
+   Model should have 4 inputs, with the following dimensions:
+
+   **input_ids** `[batch_size,seq_len]`
+
+   **position_ids** `[batch_size,seq_len]`
+
+   **attention_mask** `[batch_size,total_seq_len]`
+
+   **past_0**`[2,batch_size,16,past_seq_len,64]`
```

Comment:
Not every GPT-2 model has num_heads=16 and hidden_size_per_head=64. It might be better to add "like" in description of the dimensions.